src/applications/disability-benefits/2346/app-entry.jsx

```diff
@@ -0,0 +1,14 @@
+import '../../../platform/polyfills';
+import './sass/mdt-introduction.scss';
+
+import startApp from '../../../platform/startup';
```

Comment:
You can get rid of the extra `../../../` in the path to the `platform` folder. It's in a bunch of the files.

Reply:
@Mottie On both line 1 and 4 I should just put `./platform/` etc?